docs/pages/blog/turbo-1-5-0.mdx

```diff
@@ -0,0 +1,108 @@
+- [Development Tasks](/docs/handbook/dev)
+- [Building Apps](/docs/handbook/building-your-app)
+- [Publishing Packages](/docs/handbook/publishing-packages)
+
+## Drop the `run`
+
+<iframe className="max-w-xl h-72 sm:h-96 w-full mt-6" src="https://www.youtube.com/embed/PEgk2v6KntY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"></iframe>
+
+You can now run tasks with the Turborepo CLI using `turbo <task>`.
+
+```diff
+- turbo run build
++ turbo build
+
+
+- turbo run lint build test
++ turbo lint build test
+```
+
+If your task name conflicts with a built-in `turbo` subcommand, we'll run our subcommand instead. That means you shouldn't name your tasks things like `prune`, `run`, or `login` - since those are built-in subcommands.
+
+`turbo run <task>` will continue to work, and there are no plans to deprecate it.
+
+## Prune now supported on pnpm and yarn 2+
+
+We're delighted to announce that `turbo prune --scope=<package-name>` is now supported in pnpm and yarn 2+.
+
+You can use `turbo prune` to create a pruned version of your monorepo - including dependencies in your lockfile.
+
+As part of the handbook, we've also added a section on using `turbo prune` to [build docker images](/docs/handbook/deploying-with-docker).
+
+Check out our previous [blog on prune](/blog/turbo-0-4-0#experimental-pruned-workspaces) to learn more.
+
+## Environment variables in `turbo.json`
+
+We've introduced two new keys to `turbo.json` - `env` and `globalEnv`. These allow environment variables to be configured _separately_ from tasks:
+
+```diff filename="turbo.json"
+{
+  "globalDependencies": [
+-   "$DATABASE_URL"
+  ],
++ "globalEnv": [
++   "DATABASE_URL"
++ ],
+  "pipeline": {
+    "build": {
+      "dependsOn": [
+-       "$BUILD_ENV"
+      ],
++     "env": [
++       "BUILD_ENV"
++     ]
+    }
+  }
+}
+```
+
+`globalEnv` and `env` allow you to specify a list of environment variables _without_ `$` prefixes. This makes the configuration file significantly easier to read. Read more [in our updated docs](/docs/core-concepts/caching#altering-caching-based-on-environment-variables).
+
+To help migrate from the previous syntax, we've prepared a codemod. You can run `npx @turbo/codemod migrate-env-var-dependencies`.
+
+This work builds on the [automatic env variable detection](/blog/turbo-1-4-0) we added in 1.4.0.
+
+## Changes to `package.json` hashing
+
+The `package.json` file in each workspace is now _always_ considered an input for tasks in that workspace. This means that if you change the _definition_ for a task in `package.json`, we want to invalidate any caches from the previous definition.
+
+This also counts for the `package.json` in the root. Changes to the root `package.json` will invalidate tasks in the root workspace.
+
+This helps make Turborepo's cache a bit smarter, and less likely to trip up when task definitions change.
```

Comment:
```suggestion
This helps make Turborepo's cache a bit smarter, and less likely to trip up when task definitions change.

##  Community
Since releasing [Turborepo v1.4 in August](/blog/turbo-1-3-0), we've seen incredible adoption and community growth:

- [9.5k+ GitHub Stars](https://github.com/vercel/turborepo)
- [440k weekly NPM downloads](https://www.npmjs.com/package/turbo)
- 15 years of compute time saved through [Remote Caching on Vercel](https://vercel.com/docs/concepts/monorepos/remote-caching), saving over a 1 year per week, up 2x since July

Turborepo is the result of the combined work of all of our contributors including our core team.

This release was brought to you by the contributions of: @7flash, @afady, @alexander-young, @atilafassina, @bguedes-moz, @bobaaaaa, @brunojppb, @chris-olszewski, @DoctorJohn, @erj826, @futantan, @gsoltis, @HosseinAgha, @ivov, @jaredpalmer, @joelhooks, @knownasnaffy, @laurentlucian, @leerob, @MarceloAlves, @mattpocock, @mauricekleine, @mehulkar, @Misikir, @nareshbhatia, @nathanhammond, @pakaponk, @PhentomPT, @renovate, @ruisaraiva19, @samuelhorn, @shemayas, @shuding, @t-i-0414, @theurgi, @tknickman, @yanmao-cc, and more!

Thank you for your continued support, feedback, and collaboration to make Turborepo your build tool of choice.
```